github/resource_github_repository.go

```diff
@@ -22,6 +22,7 @@ func resourceGithubRepository() *schema.Resource {
 		Importer: &schema.ResourceImporter{
 			State: func(d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
 				d.Set("auto_init", false)
+				d.Set("archive_on_destroy", false)
```

Comment:
If a repository is imported with configuration that specifies `archive_on_destroy` to be `true`, will `terraform` perform the import and then update the resource's preference in some subsequent step?  That seems like the probable flow, but wanted to ask in case you can confirm this behaviour.

Reply:
ah yes another `plan` and `apply` will be necessary after import as the API doesn't return this info at Read time; i'm now wondering what's a better way to approach this though.. set the `archive_on_destroy` to false here at import time, or don't explicitly set a value here? in either case there will be a non-empty plan after import

Reply:
ah or we could import by `repo_name:archive_on_destroy` instead 🤔

Reply:
@anGie44 I think the value this option provides outweighs the annoyance of a `plan` / `apply` step after an import.

Perhaps we can add a log statement during the import to explain this side effect rather than modifying the `import` ID.